packages/teams-js/src/public/sharing.ts

```diff
@@ -194,4 +198,68 @@ export namespace sharing {
   export function isSupported(): boolean {
     return ensureInitialized(runtime) && runtime.supports.sharing ? true : false;
   }
+
+  /**
+   * Namespace to get the list of content shared in a context
+   *
+   * @beta
+   */
+  export namespace history {
+    /**
+     * Represents the data returned when calling {@link sharing.history.getContent}
+     *
+     * @beta
+     */
+    export interface IContentResponse {
+      /** Id of the app where the content was shared from */
+      appId: string;
+      /** Title of the shared content */
+      title: string;
+      /** Reference of the shared content */
+      contentReference: string;
+      /** Id of the thread where the content was shared. This is a UUID */
+      threadId: string;
+      /** Id of the user who shared the content. This is a UUID */
+      author: string;
+      /** Type of the shared content.
+       * For sharing to Teams stage scenarios, this value would be ShareToStage
+       */
+      contentType: string;
+    }
+
+    /**
+     * Get the list of content shared in a Teams meeting
+     *
+     * @beta
+     */
+    export function getContent(): Promise<IContentResponse[]> {
+      ensureInitialized(runtime, FrameContexts.sidePanel, FrameContexts.meetingStage);
+      if (!isSupported()) {
+        throw errorNotSupportedOnPlatform;
+      }
+
+      return sendAndHandleSdkError(
+        getApiVersionTag(sharingTelemetryVersionNumber_v1, ApiName.Sharing_History_GetContent),
+        'sharing.history.getContent',
+      ).then((contentDetails: IContentResponse[]) => {
```

Comment:
>then [](http://example.com/codeflow?start=8&length=4)

There's nothing wrong with how you wrote this but it would likely be easier to read if you use `await` instead of `.then`. Your call, though.

Reply:
Updated to use `await`.